python/example/accuracy_pitfalls.ipynb

```diff
@@ -0,0 +1,338 @@
+    "        )\n",
+    "accuracy = laplacian_scale_to_accuracy(scale, alpha)\n",
+    "print('Accuracy interval (with accuracy value {0}) contains the true mean on D_tilde with probability {1}'.format(\n",
+    "    round(accuracy, 4),\n",
+    "    np.mean([(D_tilde_mean_age >= val - accuracy) & (D_tilde_mean_age <= val + accuracy) for val in releases])))\n",
+    "\n",
+    "print('Accuracy interval (with accuracy value {0}) contains the true mean on D with probability {1}'.format(\n",
+    "    round(accuracy, 4),\n",
+    "    np.mean([(D_mean_age >= val - accuracy) & (D_mean_age <= val + accuracy) for val in releases])))"
+   ],
+   "metadata": {
+    "collapsed": false,
+    "pycharm": {
+     "name": "#%%\n"
+    }
+   }
+  },
+  {
+   "cell_type": "markdown",
+   "source": [
+    "This performance is as expected. $D$ and $\\tilde{D}$ are actually the exact same data (the maximum age in the raw data is 93, so our clamp to $[0, 100]$ does not change any values, and we know the correct $n$), so our theoretical guarantees on $\\tilde{D}$ map exactly to gaurantees on $D$.\n",
+    "\n",
+    "We now move to a scenario that is still realistic, but where the performance does not translate quite as well. In this case, we imagine that the analyst believes the data to be of size 1050 and uses the default imputation within resize so that the extra 50 elements are drawn uniformly from $\\{ 0, 1, ..., 100 \\}$.\n",
+    "\n",
+    "Note that our diagnostic testing of $\\tilde{D}$ in the code above is not trivial in this case. In the first example, we knew that clamp/resize did not change the underlying data, so we could predict exactly the data on which the DP mean would actually be calculated. This will not be true for the following examples, so we will simulate finding the true underlying mean by releasing an extra DP mean with very high epsilon."
+   ],
+   "metadata": {
+    "collapsed": false
+   }
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 50,
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "Accuracy interval (with accuracy value 0.2853) contains the true mean on D_tilde with probability 0.4725\n",
+      "Accuracy interval (with accuracy value 0.2853) contains the true mean on D with probability 0.4725\n"
+     ]
+    }
+   ],
+   "source": [
+    "# establish extra information for this simulation\n",
+    "age_lower_bound = 0.\n",
+    "age_upper_bound = 100.\n",
+    "data_size = 1050\n",
+    "\n",
+    "n_sims = 1_000\n",
+    "true_D_tilde_mean_ages = []\n",
+    "dp_D_tilde_mean_ages = []\n",
+    "\n",
+    "D_tilde = np.clip(D.age, age_bounds[0], age_bounds[1])\n",
+    "D_tilde_mean_age = np.mean(D_tilde)\n",
+    "\n",
+    "def check_scale(scale, preprocessor, dataset_distance, epsilon):\n",
+    "    return (preprocessor >> make_base_laplace(scale)).check(dataset_distance, epsilon)\n",
+    "\n",
+    "preprocessor = (\n",
```

Comment:
I think it would be clearer if this whole chain were a function of one parameter, `data_size`? Maybe `make_mean_with_data_size` (or whatever makes sense to you). Then both chains can be simplified to one function call.

Reply:
This one's up to your judgement ^^